metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -478,26 +488,30 @@ def loop_tables(
             dataset_snapshot.aspects.append(schema_metadata)
 
             # add table properties
-            properties: Dict[str, str] = {
-                "create_date": columns[-1]["create_date"],
-                "table_type": columns[-1]["table_type"],
-                "table_location": ""
-                if columns[-1]["table_location"] is None
-                else columns[-1]["table_location"],
-            }
+            default_properties = ["create_date", "table_type", "table_location"]
+            properties: Dict[str, str] = {}
+            for prop in default_properties + self.config.extra_properties:
+                if prop in columns[-1]:
+                    properties[prop] = columns[-1][prop] or ""
```

Comment:
do we need to stringify this to be safe?

```suggestion
                    properties[prop] = str(columns[-1][prop] or "")
```